tests/profile/serializers/test_metrics_serializer.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from unittest.mock import patch
+from wazimap_ng.profile.serializers.metrics_serializer import absolute_value, subindicator, sibling
+from tests.datasets.factories import IndicatorDataFactory
+
+@pytest.fixture
+def indicatordata_json():
```

Comment:
this again is the same as in the `conftest` right?

Reply:
Fixed here: 3612794e59c5e490b2950733214ffe2fb615935d